apps/api/src/app/workflows/dto/create-workflow-override-request.dto.ts

```diff
@@ -0,0 +1,37 @@
+import { IsBoolean, IsDefined, IsOptional, IsString, ValidateNested } from 'class-validator';
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { Type } from 'class-transformer';
+
+import { ICreateWorkflowOverrideRequestDto } from '@novu/shared';
+
+import { PreferenceChannels } from '../../shared/dtos/preference-channels';
+
+export class CreateWorkflowOverrideRequestDto implements ICreateWorkflowOverrideRequestDto {
+  @ApiPropertyOptional()
+  @IsString()
+  @IsOptional()
+  triggerIdentifier?: string;
+
+  @ApiPropertyOptional()
+  @IsString()
+  @IsOptional()
+  _workflowId?: string;
```

Comment:
thinking about it again i wonder if update to `workflowId` as the outside world should not be aware of the internal conversions.

```suggestion
  workflowId?: string;
```